x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/__mock__/index.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { QueryParams, SortFieldCase } from '../../../../../containers/case/types';
+
+export const useGetCasesMockState = {
+  data: {
+    cases: [
+      {
+        caseId: '3c4ddcc0-4e99-11ea-9290-35d05cb55c15',
+        createdAt: '2020-02-13T19:44:23.627Z',
+        createdBy: { username: 'elastic' },
+        description: 'Security banana Issue',
+        state: 'open',
+        tags: ['defacement'],
+        title: 'Another horrible breach',
+        updatedAt: '2020-02-13T19:44:23.627Z',
+      },
+      {
+        caseId: '362a5c10-4e99-11ea-9290-35d05cb55c15',
+        createdAt: '2020-02-13T19:44:13.328Z',
+        createdBy: { username: 'elastic' },
+        description: 'Security banana Issue',
+        state: 'open',
+        tags: ['phishing'],
+        title: 'Bad email',
+        updatedAt: '2020-02-13T19:44:13.328Z',
+      },
+      {
+        caseId: '34f8b9e0-4e99-11ea-9290-35d05cb55c15',
+        createdAt: '2020-02-13T19:44:11.328Z',
+        createdBy: { username: 'elastic' },
+        description: 'Security banana Issue',
+        state: 'open',
+        tags: ['phishing'],
+        title: 'Bad email',
+        updatedAt: '2020-02-13T19:44:11.328Z',
+      },
+      {
+        caseId: '31890e90-4e99-11ea-9290-35d05cb55c15',
+        createdAt: '2020-02-13T19:44:05.563Z',
+        createdBy: { username: 'elastic' },
+        description: 'Security banana Issue',
+        state: 'closed',
+        tags: ['phishing'],
+        title: 'Uh oh',
+        updatedAt: '2020-02-18T21:32:24.056Z',
+      },
+      {
+        caseId: '2f5b3210-4e99-11ea-9290-35d05cb55c15',
+        createdAt: '2020-02-13T19:44:01.901Z',
+        createdBy: { username: 'elastic' },
+        description: 'Security banana Issue',
+        state: 'open',
+        tags: ['phishing'],
+        title: 'Uh oh',
+        updatedAt: '2020-02-13T19:44:01.901Z',
+      },
+    ],
+    page: 1,
+    perPage: 5,
+    total: 10,
+  },
+  isLoading: false,
+  isError: false,
+  queryParams: {
+    page: 1,
+    perPage: 5,
+    sortField: SortFieldCase.createdAt,
+    sortOrder: 'desc' as QueryParams['sortOrder'],
```

Comment:
@XavierM this is here to solve this type issue in the test:
```
ERROR x-pack failed
      x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/index.test.tsx:22:25 - error TS2322: Type '{ data: { cases: { caseId: string; createdAt: string; createdBy: { username: string; }; description: string; state: string; tags: string[]; title: string; updatedAt: string; }[]; page: number; perPage: number; total: number; }; isLoading: boolean; isError: boolean; queryParams: { ...; }; filterOptions: { ...; }; }' is not assignable to type 'UseGetCasesState'.
        The types of 'queryParams.sortOrder' are incompatible between these types.
          Type 'string' is not assignable to type 'Direction'.

      22       .mockReturnValue([useGetCasesMockState, setQueryParams, setFilters]);
                                 ~~~~~~~~~~~~~~~~~~~~


      Found 1 error.
```

Reply:
we fixed it by adding type on the mock